talented-tigers/spydir/generate_images.py

```diff
@@ -0,0 +1,21 @@
+import os
+import requests
+from .models import PageImage
+
+
+def generate_images(category, n: int, query: str = "clipart") -> list:
```

Comment:
Nice to see anotations. dockstrings, clear and well formated code and a short function.